docs/api/catalog-resource.md

```diff
@@ -204,7 +204,10 @@ following scenarios:
 1. A package is **pushed**.
 1. A package is **listed**.
 1. A package is **unlisted**.
+1. A package is **deprecated**.
+1. A package is **undeprecated**.
 1. A package is **reflowed**.
+1. A package is marked **vulnerable**.
```

Comment:
I think if the vulnerability information changes a leaf is emitted too. Vulnerabilities can expand, be redacted, and edited in content IIRC. @drewgillies knows the full picture here I think.

Reply:
I think a better phrase would be `A package's **vulnerability status** has changed.` but I wanted to mirror the sentence structure of the other bullet points. I think clarity is better than my silly design preferences though so I'll update it :P

Reply:
Updated